hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -49,10 +49,10 @@ Since boot-time request mode EDN streams may be FIPS non-compliant, firmware mus
 
 #### Multiple EDNs in Boot-time Request Mode
 
-If many endpoints require boot-time entropy multiple boot-time EDNs may be required, as the EDN has a fixed maximum number of peripheral ports.
+If many endpoints require boot-time entropy, multiple boot-time EDNs may be required, as the EDN has a fixed maximum number of peripheral ports.
 Since physical entropy generation takes time, there exists a mechanism to prioritize the EDNs, to match the boot priority of each group of attached endpoints.
 To establish an order to the instantiation of each EDN, enable them one at a time.
-To ensure that the most recently enabled EDN will get next priority for physical entropy, poll the `BOOT_INST_ACK` field in the [`SUM_STS`](../data/edn.hjson#sum_sts) register before enabling the following EDN.
+To ensure that the most recently enabled EDN will get next priority for physical entropy, poll the [`MAIN_SM_STATE`](../data/edn.hjson#main_sm_state) CSR until EDN's state machine has moved past the `BootInsAckWait` state (at which point CSRNG has acknowledged the Instantiate command sent by that EDN).
 
```

Comment:
Maybe add another sentence like "After this point, the next EDN can be enabled."? Because the main point here is that the following EDN should only be enabled after the higher prio EDN got its instantiate command acked.

Reply:
Thanks, updated